### DISCUSSION TITLE
[Feature] 关于_send_group_notice 和 _get_group_notice 接口

### DISCUSSION THREAD

**Discussion by wdvxdr1123** [howmanybots/onebot#87] [想法 / 建议]
以前是有这两个接口的，在V10版本中删去了，希望能重新加回来
从手机客户端来看，发送群公告时没有`title`参数，只有`content`参数
如果能重新加入onebot标准希望能删去试验性

相关issue :
[Mrs4s/go-cqhttp #493](https://github.com/Mrs4s/go-cqhttp/issues/493)

**Comment by takayama-lily** [comment]
下划线不需要了
